libs/gltfio/src/MorphHelper.cpp

```diff
@@ -154,4 +165,21 @@ void MorphHelper::addPrimitive(cgltf_mesh const* mesh, int primitiveIndex, Entit
     }
 }
 
+void MorphHelper::addTargetNames(cgltf_mesh const* mesh, Entity entity) {
+    const auto count = mesh->target_names_count;
+    if (count == 0) {
+        return;
+    }
+
+    auto& entry = mMorphTable[entity];
+    auto& names = entry.targetNames;
+
+    assert_invariant(names.empty());
+    names = utils::FixedCapacityVector<utils::CString>::with_capacity(count);
```

Comment:
I think you can call "reserve" instead of doing a copy assignment.  Not a big deal.

Reply:
Thanks it's updated.